README.MD

```diff
@@ -32,31 +32,29 @@ Presidio _(Origin from Latin praesidium ‘protection, garrison’)_ helps to en
 
 :warning: Presidio can help identify sensitive/PII data in un/structured text. However, because Presidio is using trained ML models, there is no guarantee that Presidio will find all sensitive information. Consequently, additional systems and protections should be employed.
 
-### Demo
+### :notebook_with_decorative_cover: [Full documentation](https://microsoft.github.io/presidio)
+### :thought_balloon: [Try Presidio with your own data](https://aka.ms/presidio-demo) 
 
-[Try Presidio with your own data](https://aka.ms/presidio-demo)
 
 ## Installing Presidio
 
-1. [Using pip](docs/installation.md#using-pip)
-2. [Using Docker](docs/installation.md#using-docker)
-3. [From source](docs/installation.md#install-from-source)
+1. [Using pip](https://microsoft.github.io/presidio/installation.html#using-pip)
+2. [Using Docker](https://microsoft.github.io/presidio/installation.html#using-docker)
+3. [From source](https://microsoft.github.io/presidio/installation.html#install-from-source)
```

Comment:
The links are broken. Might be after my change with the 'use_directory_urls' flag? 
Currently gh pages has my branch:
https://github.com/microsoft/presidio/pull/582/files

Reply:
Wow good catch. Let's wait with this one then

Reply:
Fixed